packages/compass-connections/src/components/legacy-connections-modal.tsx

```diff
@@ -11,29 +11,41 @@ import {
   ModalFooter,
   Checkbox,
   Body,
+  Banner,
 } from '@mongodb-js/compass-components';
 
 import type { ConnectionStorage } from '@mongodb-js/connection-storage/renderer';
 
 const LEGACY_MODAL_STORAGE_KEY = 'hide_legacy_connections_modal';
 
-const listStyle = css({
-  listStyle: 'decimal',
-  paddingLeft: spacing[5],
-});
+const listStyle = (type: 'decimal' | 'disc') =>
+  css({
+    listStyle: type,
+    paddingLeft: spacing[4],
+  });
```

Comment:
Small suggestion, as ul and ol have semantic meaning, you probably can use this to your advantage and rewrite this as static to avoid calling css inside render:

```suggestion
const listStyle = css({
    paddingLeft: spacing[4],
    'ol&': {
      listStyle: 'decimal'
    },
    'ul&': {
      listStyle: 'disc'
    }
  })
```

Reply:
thanks, i just made some more ux fixes as I synced with design.